config/schedule.rb

```diff
@@ -45,3 +45,7 @@ def local(time)
     rake "reports:telemedicine"
   end
 end
+
+every 5.minutes, roles: [:cron] do
```

Comment:
All queries on a production-sized dataset run in under 500ms. This should be safe to run on production several times.